civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -106,15 +107,19 @@
 @import "components/**/*";
 @import "drupal/**/*";
 
-@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/style/button-groups";
 
+// Imports from other styles
+// --------------------------------------------------
+@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/style/button-groups";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/custom-select";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/date-picker";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/text-angular";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/wysiwyg";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/style/ui-select";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/variants/modals-xs-screen";
 
+@import "SCSSROOT/org.civicrm.bootstrapcivihr/scss/components/outside-namespace/user-menu";
```

Comment:
why the extra space around this particular import and none of the others?

Reply:
dunno, just a visual way to separate the extensions. You think it's pointless?

Reply:
No, it is good actually. I didn't notice the different extensions at first, but now that you mention it it's clear.